css/main.css

```diff
@@ -168,8 +168,14 @@ nav .user-greet .user-greet-msg {
 
 .social-media-logo {
   vertical-align: middle;
-  width: 15px;
-  height: 15px;
+  width: 35px;
+  height: 35px;
+}
+
+.social-media-logo:hover{
```

Comment:
Nice idea to add a subtle sizing change to reflect you can interact with the element
 
The issue is that the change looks too 'abrupt' as there's no smooth transitioning applied on them. Can we add that please?

Reply:
@KhushalAbrol , could you address the above comment please